apps/src/signUpFlow/FinishTeacherAccount.tsx

```diff
@@ -46,6 +48,22 @@ const FinishTeacherAccount: React.FunctionComponent<{
     );
   };
 
+  const sendFinishEvent = (): void => {
+    const hasSchool = !!document.querySelector(
+      'select[name="user[school_info_attributes][school_id]"]'
+    );
+    analyticsReporter.sendEvent(
+      EVENTS.SIGN_UP_FINISHED_EVENT,
+      {
+        'user type': 'teacher',
+        'has school': hasSchool,
+        'has marketing value selected': true,
```

Comment:
Both for this one and the student equivalent, is the `has marketing value selected` representing the checkbox for receiving marketing emails? If so, shouldn't that be the boolean variable rather than a constant `true`?